src/controllers/user.js

```diff
@@ -68,22 +68,21 @@ export const getAll = async (req, res) => {
 }
 
 export const updateById = async (req, res) => {
-  const paramsId = Number(req.params.id)
-  const { id } = req.user
-  const foundUser = await User.findById(paramsId)
+  const { firstName, lastName } = req.body
+  try {
+    validation.update(firstName, lastName)
```

Comment:
A `try/catch` should contain as little lines of code as possible (ideally, one line). Try to refactor so that you are only catching the lines of code that can actually throw something